src/scss/blocks/_site-header.scss

```diff
@@ -138,6 +138,7 @@
 
   .site-header[class*='expanded'] web-navigation-drawer {
     opacity: 0;
+    pointer-events: none !important;
```

Comment:
FWIW: The reason I used `!important` here is because a few lines down it has `pointer-events` explicitly set. I don’t want to remove the line below, as I can’t gauge which side-effects that would cause.